backtrace-library/src/main/java/backtraceio/library/services/SummedEventsHandler.java

```diff
@@ -0,0 +1,68 @@
+package backtraceio.library.services;
+
+import android.content.Context;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentLinkedDeque;
+
+import backtraceio.library.interfaces.Api;
+import backtraceio.library.models.BacktraceMetricsSettings;
+import backtraceio.library.models.metrics.EventsPayload;
+import backtraceio.library.models.metrics.SummedEvent;
+import backtraceio.library.models.metrics.SummedEventsPayload;
+
+public class SummedEventsHandler extends BacktraceEventsHandler<SummedEvent> {
+
+    private final static transient String LOG_TAG = SummedEventsHandler.class.getSimpleName();
+
+    private final static String urlPrefix = "summed-events";
+
+    public SummedEventsHandler(Context context, Map<String, Object> customAttributes,
+                               Api api, final BacktraceHandlerThread backtraceHandlerThread, BacktraceMetricsSettings settings) {
+        super(context, customAttributes, api, backtraceHandlerThread, urlPrefix, settings);
+    }
+
+    @Override
+    public void sendStartupEvent(String eventName) {
+        events.addLast(new SummedEvent(eventName));
+        send();
+        events.clear();
```

Comment:
Are you sure it won't clean up events in the submission queue or even before send starts sending data?